src/main/java/baseball/Validator.java

```diff
@@ -0,0 +1,23 @@
+package baseball;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Validator {
+    public static void validateInput(String input) {
+        if (input.length() != 3) {     // 길이가 3이 아닐떄
+            throw new IllegalArgumentException();
```

Comment:
```suggestion
            throw new IllegalArgumentException("3자리 숫자를 입력해야합니다.");
```
게임을 하는 사용자는 2자리를 입력하고 게임이 종료되면, 숫자 입력을 잘못해서 게임이 종료된 사실을 알 수 있을까?
로그를 보는 개발자도 명확한 메세지가 없다면, 예외가 발생한 지점을 찾아봐야 어디서 문제가 생겼는지 알 수 있을 거야.
예외를 발생시킨다면, 명확한 예외 메세지도 함께 작성해주면 좋을 것 같다!